src/userSurvey.js

```diff
@@ -49,12 +49,19 @@ permissions.check({
 let storageSpace = null;
 
 /**
- * Whether a survey has already been created. This module
- * currently only supports one survey per study.
+ * The ID of the survey reminder timeout (is null if there 
+ * is no such timeout).
+ * @type {number|null}
+ * @private
+ */
+let reminderTimeoutId = null;
+
+/**
+ * Whether listeners for this module have already been registered.
  * @type {boolean}
  * @private
  */
-let createdSurvey = false;
+let listenersRegistered = false;
 
 // Module-wide variables for a survey, set in createSurvey
```

Comment:
Would be good to provide documentation for each of the module-wide variables. Would also make sense to match the object type used as a parameter for `setSurvey` and in storage.

Reply:
Added documentation but not matching the object type used as a parameter for `setSurvey` because `lastSurveyRequest` is not passed in as a parameter and some of the properties of the passed in `options` parameter do not need to be available module-wide.